gui/src/main/java/com/devonfw/ide/gui/MainController.java

```diff
@@ -0,0 +1,132 @@
+package com.devonfw.ide.gui;
+
+import java.io.File;
+import java.nio.file.Path;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+
+import com.devonfw.tools.ide.context.IdeStartContextImpl;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import com.devonfw.tools.ide.log.IdeLogListenerBuffer;
+import com.devonfw.tools.ide.log.IdeSubLoggerOut;
+
+/**
+ *
+ */
+public class MainController {
+
+  @FXML
+  private ComboBox<String> selectedProject;
+  @FXML
+  private ComboBox<String> selectedWorkspace;
+  @FXML
+  private Button androidStudioOpen;
+  @FXML
+  private Button eclipseOpen;
+  @FXML
+  private Button intellijOpen;
+  @FXML
+  private Button vsCodeOpen;
+
+  private final String directoryPath = "C:\\projects\\";
```

Comment:
Please do not hardcode such Windows specific paths.
Use `IdeContext` and `IDE_ROOT` variable instead.